seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumAsyncJSRunner.scala

```diff
@@ -26,7 +26,7 @@ class SeleniumAsyncJSRunner(browserProvider: SeleniumBrowser,
   }
 
   override def stop(): Unit = synchronized {
-    if (!keepAlive) {
+    if (!keepAlive || code.name != "testSlave.js") {
```

Comment:
That's a bit fragile. There's no guarantee that this name will stay `testSlave.js` forever. :-s